packages/components/radio/src/radio.vue

```diff
@@ -8,6 +8,7 @@
       ns.is('checked', modelValue === label),
       ns.m(size),
     ]"
+    v-bind="labelAttrs"
```

Comment:
```suggestion
      rawAttrs.class,
    ]"
    :style="containerStyle"
```

Reply:
This will invalidate the event.

Reply:
If you need to split events, you can refer to the useAttrs in hooks.

It is better to match key ​​instead of value.